mithril-common/src/chain_observer/model.rs

```diff
@@ -1,20 +1,49 @@
 use anyhow::anyhow;
-use pallas_primitives::conway::PlutusData;
-use serde::{Deserialize, Serialize};
+use anyhow::Context;
+use serde::Serialize;
 use serde_json::Value;
 use std::collections::HashMap;
 use strum::{Display, EnumDiscriminants};
 use thiserror::Error;
 
 use crate::{StdError, StdResult};
 
-/// [Datum] represents an inline datum from UTxO.
-#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, PartialOrd, Ord, Clone)]
-#[serde(rename_all = "lowercase")]
-pub struct Datum(pub PlutusData);
+cfg_fs! {
```

Comment:
Why is this macro require? does it have to do with fs = filesystem? I don't see any access to the file system in this block.